modules/core/src/test/scala/org/tessellation/domain/statechannel/StateChannelServiceSuite.scala

```diff
@@ -58,6 +58,9 @@ object StateChannelServiceSuite extends MutableIOSuite {
     val validator = new StateChannelValidator[IO] {
       def validate(output: StateChannelOutput) =
         IO.pure(failed.fold[StateChannelValidator.StateChannelValidationErrorOr[StateChannelOutput]](output.validNec)(_.invalidNec))
+
+      def validateHistorical(output: StateChannelOutput) =
```

Comment:
Looks exactly the same as `validate` what is the difference?

Reply:
Though that's suite so that may be ok.

Reply:
I've assigned the `validate` fn in both places in tests